### PR TITLE
7627: add contribution and style guide section

### PR DESCRIPTION
**Pull request by ziedAb**

### CODE REVIEW

File: content/styleguide/styleguide.rst [hunk 1]
```diff
@@ -0,0 +1,623 @@
...
+.. _other-title-considerations:
+      
+Other titling considerations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Do not put step numbers in section titles.
+- Readers skim. Section titles should be clear and provide information.
+
+  
+.. _writing-about-code:
```
Comment (hooverlunch) [inline content/styleguide/styleguide.rst]: I don't think we will have any code in our documentation. I think we can delete everything from here down.

File: content/styleguide/styleguide.rst [hunk 2]
```diff
@@ -0,0 +1,623 @@
...
+"Same"
+~~~~~~~~~
+
+*Same*, when used as an impersonal pronoun, is non-standard in Modern American English. It should be avoided.
+
+.. rubric:: Wrong
+
+.. code-block:: rst
+
+  ODK Collect is an Android app. The same can be used for...
```
Comment (hooverlunch) [inline content/styleguide/styleguide.rst]: Can you replace any mention of ODK with something else please? There are multiple throughout the file.

File: content/styleguide/styleguide.rst [hunk 3]
```diff
@@ -0,0 +1,623 @@
...
+~~~~~~~~~~~~~~~~
+
+For all HTML samples, tag names and attribute names should be ``all lowercase``. 
+
+Newly-written XML examples should also be ``all lowercase``.
+
+XML examples that show actual code generated by tools in the ODK ecosystem should replicate that code exactly, regardless of its capitalization practice.
+
+
+.. endignore
```
Comment (hooverlunch) [inline content/styleguide/styleguide.rst]: We need to put some attribution in this document. The original says: "This document is licensed under a Creative Commons Attribution 4.0 International License." So we just need to put where it came from and we can reuse and modify it.

**Comment by ziedAb** [comment]
I have done the above changes on both style guide and contributing sections.
ODK also updated the contributing section yesterday by dividing it into subsections.

**Pull request merged**